plugins/kubernetes-backend/src/service/KubernetesProxy.test.ts

```diff
@@ -15,38 +15,37 @@
  */
 import 'buffer';
 
-import { errorHandler, getVoidLogger } from '@backstage/backend-common';
```

Comment:
oh I think some unrelated changes snuck in somehow!

Reply:
yup @jhaals sorry was trying to see whats causing the error on the Kube tests, they seem to be happening across all branches that have pulled from main today... 😢 

but reverted

Reply:
but the test errors are blocking me and I can't replicate locally